src/olympia/versions/models.py

```diff
@@ -51,6 +51,14 @@ def get_queryset(self):
             qs = qs.exclude(deleted=True)
         return qs.transform(Version.transformer)
 
+    def with_all_activity(self):
+        from olympia.devhub.models import VersionLog
+        versionlog_qs = (
+            VersionLog.objects.no_cache().order_by('created')
+            .select_related('activity_log', 'version'))
+        return self.get_queryset().prefetch_related(models.Prefetch(
```

Comment:
Can we have the prefetch on it's own line so that it looks nicer?

```python
prefetch = models.Prefetch(...)
return self.get_queryset().prefetch_related(prefetch)
```